### PR TITLE
SNO-47 Add index time log option and update index output data

### PR DESCRIPTION
**Pull request by caseylitton**
This PR updates the Indexer.update_object function so it returns more information to update_objects.  This is the only way to get info back from update_object when running MPIndexer.update_objects since it's in sub processes.

Also this PR adds an index_logger class that is imported and added to indexer.py:Indexer class.  The logger is added to all indexers, viz, region, and primary but is only turned on in the primary indexer.  So includeme was updated in indexer.py and mpindexer.py.

I will add this to the ini configs so we can turn it on/off without changing repo code.

### CODE REVIEW

File: src/snovault/elasticsearch/index_logger.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        '''Close all log handlers'''
+        for handler in self._the_log.handlers:
+            handler.close()
+            self._the_log.removeHandler(handler)
+
+    def _get_log(self):
+        if self._do_log:
+            file_name = "{}-{}.log".format(
+                self.log_name,
+                str(int(time.time() * 10000000)),
```
Comment (hitz) [inline src/snovault/elasticsearch/index_logger.py]: this is converting microsec-> sec?   Probably should have a comment here.

File: src/snovault/elasticsearch/index_logger.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            return log
+        return None
+
+    def _reset_log(self):
+        '''
+        Close handlers and Get new log
+
+        * Logger gets logs name so we call twice to clear and get a new one
+        '''
+        if self._do_log:
```
Comment (hitz) [inline src/snovault/elasticsearch/index_logger.py]: this seems super clunky but I guess you have a good reason?

Comment (caseylitton) [reply src/snovault/elasticsearch/index_logger.py]: eh...It's clunky.  I didn't like it.  I used the flag in the logger class to limit the indexer code changes.  Now that it works maybe I can find a better solution.

File: src/snovault/elasticsearch/index_logger.py (hunk 1 above)
Comment (caseylitton) [reply src/snovault/elasticsearch/index_logger.py]: Well yes and no, It's just a way to keep timestamps on the file.  I can add a comment anyhow.

**Comment by caseylitton** [comment]
Things to update, 

add time.time() for end time
add pid to log
backoff time
look through embedd stack

**Comment by caseylitton** [comment]
reopen in future

**Pull request closed without merging**